### PR TITLE
Update importing doc with GitHubBox and Browser Extension options

### PR DESCRIPTION
**Pull request by garethx**
Making this a PR while we wait for the new firefox extension page to be approved.

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/4092/commits/1a600d1e11d1b94c7f8c67a7b5248217271eb10d is at https://pr4092.build.csb.dev/s/new.

File: packages/homepage/content/docs/2-importing.md [hunk 1]
```diff
@@ -19,20 +19,35 @@ and begin creating your own sandbox.
...
-### Setting inference
+### Using GitHubBox.com
+An easy way to import a repo to CodeSandbox via URL is with GitHubBox.com. Append 
+'box' to your github.com URL in between 'hub' and '.com' and it will redirect to 
+CodeSandbox.
+
+Here's an example:
+
+Change the GitHub URL:
+[https://github.com/reduxjs/redux/tree/master/examples/todomvc](https://github.com/reduxjs/redux/tree/master/examples/todomvc).
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-importing.md:38]: You can just use the link here
```suggestion
https://github.com/reduxjs/redux/tree/master/examples/todomvc.
```

File: packages/homepage/content/docs/2-importing.md [hunk 2]
```diff
@@ -19,20 +19,35 @@ and begin creating your own sandbox.
...
+
+To:
+[https://githubbox.com/reactjs/redux/tree/master/examples/todomvc](https://githubbox.com/reactjs/redux/tree/master/examples/todomvc).
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-importing.md:41]:
```suggestion
https://githubbox.com/reactjs/redux/tree/master/examples/todomvc.
```

File: packages/homepage/content/docs/2-importing.md [hunk 3]
```diff
@@ -19,20 +19,35 @@ and begin creating your own sandbox.
...
-[https://codesandbox.io/s/github/reactjs/redux/tree/master/examples/todomvc](https://codesandbox.io/s/github/reactjs/redux/tree/master/examples/todomvc).
+GitHub repository. We support custom branches and subdirectories.
 
 The imported repository will always stay up to date with your latest commits.
 This means that any change to the GitHub repository will be reflected
-immediately to the sandbox.
+immediately in the sandbox.
 
-### Setting inference
+### Using GitHubBox.com
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-importing.md:30]: Prettify document

```suggestion
### Using GitHubBox.com

```

File: packages/homepage/content/docs/2-importing.md [hunk 4]
```diff
@@ -19,20 +19,35 @@ and begin creating your own sandbox.
...
+[https://github.com/reduxjs/redux/tree/master/examples/todomvc](https://github.com/reduxjs/redux/tree/master/examples/todomvc).
+
+To:
+[https://githubbox.com/reactjs/redux/tree/master/examples/todomvc](https://githubbox.com/reactjs/redux/tree/master/examples/todomvc).
+
+The result is we take the last part of the url (everything after github.com) and 
+use it in our importer at codesandbox.io/s/github/, adding the repo to CodeSandbox.
 
-We infer sandbox settings based on several files in a repository.
+### Using a Browser Extension
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-importing.md:46]:
```suggestion
### Using a Browser Extension

```

File: packages/homepage/content/docs/2-importing.md [hunk 5]
```diff
@@ -19,20 +19,35 @@ and begin creating your own sandbox.
...
+We have browswer extensions for [Chrome](https://chrome.google.com/webstore/detail/open-in-codesandbox/hdidglkcgdolpoijdckmafdnddjoglia) and [Firefox](https://addons.mozilla.org/en-GB/firefox/addon/codesandbox/), which add an 'Open in CodeSandbox' button to GitHub repo pages. This makes it easy to import existing projects from GitHub in to CodeSandbox.
+
+### Setting inference
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-importing.md:49]:
```suggestion
### Setting inference

```

**Comment by garethx** [comment]
Firefox extension has now been approved

**Pull request merged**